buzzmobile/sense/lidar/lidar.py

```diff
@@ -1,28 +1,55 @@
 import cv2
 import numpy as np
-
 import time
 
-image_height_px = 800
-image_width_px = 800
-image_height_m = 8
-image_width_m = 8
-pixels_per_m = 100
+from cv_bridge import CvBridge, CvBridgeError
+from sensor_msgs.msg import Image
+from sensor_msgs.msg import LaserScan
+import rospy
+
+import sys
+sys.path.append('../../include')
```

Comment:
I think this works as a stopgap though. So I'll just make an issue for that and resolve this.